api/config/routes.rb

```diff
@@ -166,6 +166,9 @@
 
         resources :menus, only: %i[index]
         get '/menus/:location', to: 'menus#show', as: :menu
+
+        resources :cms_pages, only: %i[index], path: '/pages'
+        get '/pages/:slug', to: 'cms_pages#show', as: :page
```

Comment:
```suggestion
        resources :cms_pages, only: %i[index]
        get '/cms_pages/:slug', to: 'cms_pages#show', as: :page
```

this way we're avoiding any name clashing